_content/lorawan/frequency-plans.md

```diff
@@ -61,6 +61,8 @@ Downlink:
 7. **926.9** - SF7BW500 to SF12BW500
 8. **927.5** - SF7BW500 to SF12BW500
 
+Note that The Things Network uses 2nd Sub-Band only (channels 8,9,10,11,12,13,14,15,65). If your devices use OTAA and ADR, this will be configured automatically when they join (expect some join retries and the first ~8 packets to fail). If your devices use ABP, you'll need to program the specific channels into the devices in order to make them work with TTN.
```

Comment:
- Use `>` indentation
- FSB 2, and only on the public community network (2x)
- Channels `[8..15, 65]` or `8 to 15 and 65`
- Why would the first 8 packets fail?

Reply:
>An unpersonalized device will try to join using the whoole 64+8 channels, so expect ~8 join retials before matching one join in the 2nd sub-band.

That depends on the channel hopping strategy on join. Best practices are to try subband after subband with a random channel, not trying all channels in the first subband first.

Reply:
yes, that's why you can expect up to 8 joins to get joined, you need not wait up to 56 joins

Reply:
According to my beer mat calculation; with all join requests resulting in a join accept, the expected number of retries is 1 (FSB 1 attempt fails).